docs/index.html

```diff
@@ -3,142 +3,101 @@
 title: Docs
 group: Docs
 ---
-<div id="content">
-<h2>Documentation for i3 v4.19</h2>
-
-<p>
-One of i3’s goals is good documentation. The documents which you will find
-below will hopefully answer all your questions. If you have any corrections or
-suggestions please let us know!
-</p>
-
-<div class="docs">
-<h2>For users</h2>
-
-<p style="font-size: 100%; background-color: #444; border-radius: 5px; padding: 0.5em; width: 90%">
-<a href="/docs/userguide.html" style="font-size: 125%"><strong>User’s Guide</strong></a><br>
-Introduction and reference. Read this!
-</p>
-
-<p>
-<a href="/conduct"><strong>Code of Conduct</strong></a><br>
-Outlines acceptable behavior within the i3 projects.
-</p>
-
-<p>
-<a href="/docs/layout-saving.html"><strong>Layout saving/restoring</strong></a><br>
-Explains how to save a layout and restore it in a new i3 session.
-</p>
-
-
-<p>
-<a href="/docs/multi-monitor.html"><strong>Multi-monitor</strong></a><br>
-Interesting for users of the nVidia driver.
-</p>
-
-<p>
-<a href="/docs/debugging.html"><strong>Debugging i3</strong></a><br>
-Explains you how to enable debug logging.
-</p>
-
-<p>
-<a href="/docs/wsbar.html"><strong>External workspace bars</strong></a><br>
-About bar programs such as i3bar or dzen2.
-</p>
-
-<p>
-<a href="/docs/refcard.html"><strong>i3 reference card</strong></a><br>
-Might be useful to memorize i3’s shortcuts.
-</p>
-</div>
-
-<div class="docs">
-<h2>For developers</h2>
-
-<p style="padding-top: 1em">
-<a href="/docs/hacking-howto.html"><strong>Hacking Howto</strong></a><br>
-Helps you if you want to get into i3’s source code.
-</p>
-
-<p>
-<a href="/docs/debugging.html"><strong>Debugging i3</strong></a><br>
-Explains you how to enable debug logging.
-</p>
-
-<p>
-<a href="/docs/ipc.html"><strong>Inter process communication (IPC interface)</strong></a><br>
-Read this if you want to talk to i3 within your script.
-</p>
 
-<p>
-<a href="/docs/testsuite.html"><strong>i3 testsuite</strong></a><br>
-Makes you able to read and write i3 testcases.
-</p>
-
-<p>
-<a href="/docs/i3bar-protocol.html"><strong>i3bar protocol</strong></a><br>
-Documents the JSON based protocol which i3bar uses.
-</p>
-
-</div>
+<h2>Documentation for i3 v4.19</h2>
 
-<br style="clear: both">
+<p>One of i3’s goals is good documentation. The documents which you will find below will hopefully answer all your questions. If you have any corrections or suggestions please let us know!</p>
+
+<section class="columns">
+    <div class="docs">
+        <h2>For users</h2>
+        <div class="item highlight">
+            <a href="/docs/userguide.html">User’s Guide</a>
+            <p>Introduction and reference. Read this!</p>
+        </div>
+        <div class="item">
+            <a href="conduct">Code of Conduct</a>
```

Comment:
This link still isn’t absolute.

Could you run a link checker on your working copy to make sure we don’t accidentally introduce any broken links? As you can see, it’s easy to miss them :)

Reply:
See a89ade7

I ran a linkchecker and only got two 404s. https://i3wm.org/downloads/RELEASE-NOTES-3.a-bf2.txt and https://i3wm.org/downloads/RELEASE-NOTES-3.a.txt, both of which also 404 on the live site. I also got some hits on SVGs on the refcard, but I can look to update the refcard in a separate PR. It seems to render identical to the live version.